File: Iniciante/URI 1044.py
a = None
b = None
lista = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista = read_line().split(" ")
a = int(lista[0])
b = int(lista[1])

if a % b == 0 or b % a == 0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")

File: Iniciante/URI 1044.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        int a = int.Parse(input[0]);
        int b = int.Parse(input[1]);
        if (a % b == 0 || b % a == 0) {
            Console.WriteLine("Sao Multiplos");
        } else { 
            Console.WriteLine("Nao sao Multiplos");
        }
    }
    
}